api/v1alpha1/workflow_webhook.go

```diff
@@ -76,8 +76,11 @@ var _ webhook.Validator = &Workflow{}
 // ValidateCreate implements webhook.Validator so a webhook will be registered for the type
 func (w *Workflow) ValidateCreate() error {
 
-	if w.Spec.DesiredState != "proposal" {
-		return fmt.Errorf("desired state must start in 'proposal'")
+	if w.Spec.DesiredState != StateProposal.String() {
+		return fmt.Errorf("desired state must start in %s", StateProposal.String())
+	}
+	if w.Spec.Hurry == true {
+		return fmt.Errorf("The hurry flag may not be set on creation")
```

Comment:
Error strings should not be capitalized https://github.com/golang/go/wiki/CodeReviewComments#error-strings